lib/npg_tracking/daemon/elembiostaging.pm

```diff
@@ -0,0 +1,94 @@
+package npg_tracking::daemon::elembiostaging;
+
+use Moose;
+use namespace::autoclean;
+use Readonly;
+
+extends 'npg_tracking::daemon';
+
+our $VERSION = '0';
+
+Readonly::Scalar my $SCRIPT_NAME => q[elembio_staging_area_monitor];
+Readonly::Array  my @STAGING_AREAS => qw(/lustre/scratch120/elembio/staging);
+
+override 'daemon_name'  => sub { return $SCRIPT_NAME; };
+override 'command'      => sub {
+  ## no critic (CodeLayout::ProhibitParensWithBuiltins)  
+  return sprintf '%s %s',
+                 $SCRIPT_NAME,
+                 join(q[ ], map { "--staging_area $_"} @STAGING_AREAS);
```

Comment:
I would also add `--verbose` to get messages at RunFolder logger level

Reply:
my bad, the current behaviour will use more verbosity anyway with no option specified